pkg/store/store.go

```diff
@@ -37,9 +38,44 @@ func NewAtomixStore(client atomix.Client) (Store, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &atomixStore{
+	store := &atomixStore{
 		objects: objects,
-	}, nil
+		relations: relationMaps{
+			targets: map[string][]topoapi.ID{},
+			sources: map[string][]topoapi.ID{},
+			lock:    sync.RWMutex{},
+		},
+	}
+
+	// watch the atomixStore for changes
```

Comment:
I'd also suggest to move this watch-related code into a separate function, e.g. startWatchingChanges or something of the sort. I know it's largely cosmetic at this point, but payback comes down the line where someone else comes to maintain this.